inc/namespace.php

```diff
@@ -32,10 +30,10 @@ function bootstrap() {
 		register_shutdown_function( __NAMESPACE__ . '\\on_shutdown' );
 	}
 
-	$current_errror_handler = set_error_handler( function () use ( &$current_errror_handler ) { // @codingStandardsIgnoreLine
+	$current_error_handler = set_error_handler( function () use ( &$current_error_handler ) { // @codingStandardsIgnoreLine
```

Comment:
Unrelated to this PR but is the ampersand still needed? Think this will break in PHP 7.3

Reply:
Yes; pass-by-ref is required in `use` statements (unlike function arguments) as it can alter the parent scope's variable.